packages/esbuild-why/report.js

```diff
@@ -0,0 +1,10 @@
+const isValidFilename = require('valid-filename')
```

Comment:
The check is too simple to use another dependency. Let’s just copy regexp

https://github.com/sindresorhus/filename-reserved-regex/blob/main/index.js

Reply:
I pulled in the code directly. Added attribution